ui/tests/playwright/ClusterOverview.test.tsx

```diff
@@ -0,0 +1,60 @@
+import { page } from "../../jest.setup";
+describe("Cluster Overview page", () => {
+  test("Cluster overview page should display correctly", async () => {
+    await page.goto(
+      "https://console.amq-streams-ui.us-east.containers.appdomain.cloud/kafka/j7W3TRG7SsWCBXHjz2hfrg/overview"
+    );
+    await page.waitForLoadState("networkidle");
+    expect(await page.innerText("body")).toContain("Cluster overview");
+    expect(await page.innerText("body")).toContain(
+      "Key performance indicators and important information regarding the Kafka cluster."
+    );
+    const noMessagesText = await page.$eval('span[id="no-messages"]', (span) =>
+      span?.textContent?.trim()
+    );
+    expect(noMessagesText).toBe("No messages");
+    //const page = await page.innerText('div[class="pf-v5-l-flex pf-m-justify-content-center pf-m-wrap pf-m-nowrap-on-sm pf-m-gap"]');
+    expect(await page.innerText("body")).toContain("kafka1");
+    expect(await page.innerText("body")).toContain("Ready");
+    expect(await page.innerText("body")).toContain("3/3");
```

Comment:
You can't really have tests for the values without controlling the environment. These values will be invalid the moment the cluster is used for something, like for the demo data generation, that will change the amount of disk space used/available.

It's enough to check for the labels for the moment.